LEGO1/tglvector.h

```diff
@@ -0,0 +1,277 @@
+#ifndef TGLVECTOR_H
+#define TGLVECTOR_H
+
+#include "math.h" // ??? sin() in  RotateAroundY()
+
+#include <stddef.h> // offsetof()
+
+namespace Tgl
+{
+
+namespace Constant
+{
+const float Pi = 3.14159265358979323846;
```

Comment:
does MSVC's own library have a pi constant? especially in math.h

Reply:
Perhaps, the code uses several slightly different versions of Pi, in different places.  And also a few other constants derived from that, like pi/180 and 180/pi for conversion.